app/web/src/store/func/funcs.store.ts

```diff
@@ -147,23 +147,9 @@ export const useFuncStore = () => {
             return undefined;
           },
 
-        proprForInputSocketId:
+        inputSocketForId:
```

Comment:
This was the implicit `InternalProvider` to `Prop` conversion getter. It is now dead.